xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/macros.vm

```diff
@@ -73,6 +73,7 @@
  *#
 #macro(editActionButton $action $resourceIdentifier $class)
 #submitButton("action_${action}", $services.localization.render("core.shortcuts.edit.${resourceIdentifier}"), $services.localization.render($resourceIdentifier), $class)
+<input type="hidden" name="xaction" value="$action" />
```

Comment:
You don't know who calls this macro so I would escape the XML in the attribute value to be safe.

Reply:
Does wrapping the input tag with {{html clean="false"}}...{{/html}} ensure the safe escaping of the $action value?

Reply:
That's not what the html macro is about, it's job is to tell the renderer that there is some HTML to insert as is so by definition it actually does not do anything and provent other to do anything. The need here is to escape the value of $action so that it does not break the `<input>`. For that you can use a Velocity tool, here it's $escapetool.xml($action).